ports/bananaguy2/Bananaguy 2.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+get_controls
+
+GAMEDIR=/$directory/ports/bananaguy2/
+CONFDIR="$GAMEDIR/conf/"
+
+# Ensure the conf directory exists
+mkdir -p "$GAMEDIR/conf"
+
+# Set the XDG environment variables for config & savefiles
+export XDG_CONFIG_HOME="$CONFDIR"
+export XDG_DATA_HOME="$CONFDIR"
+
+cd $GAMEDIR
+
+# Unpack bananaguy2.zip using unzip if it exists
+if [[ -f "gamedata/bananaguy2.zip" ]]; then
+  if "libs/7za" x "gamedata/bananaguy2.zip" -o"gamedata"; then
```

Comment:
Either that or you might need to leave it unzipped, and use `python3 tools/build_data.py`, this will split the file into 50mb chunks that will be recombined when the port zip is built.